### ISSUE TITLE
phpcbf PSR12 can't fix multi-line function call inside a string concat statement

### ISSUE THREAD

**Issue by stdex** [squizlabs/PHP_CodeSniffer#2976]
Hello.

PHP_CodeSniffer version 3.5.5 (stable)

Call:
phpcbf.bat --report=full -s -vv --standard=psr12

Problem with code:

```php
<?php

class C
{

    public function m()
    {
        $t = '1' . (true ? '
2' : '
3') . '4' . f(
        '5',
        );
    }
}
```

```
        => Fixing file: 2/2 violations remaining [made 49 passes]...    * fixed 2 violations, starting loop 50 *
        **** Generic.WhiteSpace.ScopeIndent:1479 has possible conflict with another sniff on loop 48; caused by the following change ****
        **** replaced token 53 (T_WHITESPACE on line 11) "    '5'" => "        '5'" ****
        **** ignoring all changes until next loop ****
        => Fixing file: 0/2 violations remaining [made 50 passes]...    * fixed 0 violations, starting loop 51 *
DONE in 482ms
        => File was overwritten

PHPCBF RESULT SUMMARY
----------------------------------------------------------------------
FILE                                                  FIXED  REMAINING
----------------------------------------------------------------------
1.php           FAILED TO FIX
----------------------------------------------------------------------
A TOTAL OF 0 ERRORS WERE FIXED IN 1 FILE
----------------------------------------------------------------------
PHPCBF FAILED TO FIX 1 FILE
----------------------------------------------------------------------
```
